content/en/docs/tasks/configure-pod-container/user-namespaces.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Use a user namespace with a stateless pod
+reviewers:
+content_type: task
+weight: 160
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+This page shows how to configure a user namespace for stateless pods. This
+allows to isolate the user running inside the container from the one in the
+host. A process running as root in a container can run as a different (non-root)
+user in the host.
```

Comment:
```suggestion
host node. A process running as root in a container can actually run as a different,
non-root user as seen by the node's host OS.
```

Reply:
I've changed it to something that seems even better, from man 7 user_namespaces:

```
In particular, a process can have a
normal unprivileged user ID outside a user namespace while at the same time
having a user ID of 0 inside the namespace; in other words, the process has full
privileges for operations inside the user namespace, but is unprivileged for
operations outside the namespace.
```